.github/workflows/release-binaries.yml

```diff
@@ -20,6 +20,9 @@ jobs:
           - os: ubuntu-latest
             target: linux-musl
             arch: aarch64
+          - os: macos-14
+            target: apple-darwin
+            arch: aarch64
```

Comment:
nit: Ubuntu seems to call its arm-based version `aarch64`. However, the apple silicon mac seems to call itself `arm64`. I think that will require a larger change in the build heirarchy, not really required. Just an FYI.

Reply:
On our build environment (Nixpkgs) it's aarch64, which is the official nomenclature.